pom.xml

```diff
@@ -13,6 +13,12 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <jenkins-test-harness.version>1.625</jenkins-test-harness.version>
```

Comment:
The JTH comes from the plugin POM, so without overriding it, it shouldn't be bringing JTH `1.424`, right? I mean, I recall making this exact same change mane times and not having to override the JTH.